salt/modules/drbd.py

```diff
@@ -1,13 +1,47 @@
 # -*- coding: utf-8 -*-
 '''
-DRBD administration module
+Module to provide DRBD functionality to Salt
+
+.. versionadded:: pending
```

Comment:
since this module is not new to neon can you remove the versionadded here and add `.. versionadded:: neon` to each of the new functions's doc string in the module?

Reply:
Add to all new function of module.